masonry/src/widget/widget_state.rs

```diff
@@ -170,6 +170,20 @@ impl WidgetState {
         }
     }
 
+    pub(crate) fn root(id: WidgetId, size: Size) -> WidgetState {
```

Comment:
I find this `root` thing quite confusing, because my general understanding is that the root widget has its own state.

Reply:
It's useful for lifetime purposes; but yeah, I think at this point it can probably be considered vestigial. I might remove it once other passes are implemented.